LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is not a part of this travis

Reply:
Well I guess it was pulled from master somehow and got here. I'll delete it